<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadClassFromWebWidget</class>
 <widget class="QWidget" name="LoadClassFromWebWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>导入课程</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SIMS_res.qrc">
    <normaloff>:/SIM_System_Pics/resources/icon_login.png</normaloff>:/SIM_System_Pics/resources/icon_login.png</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>591</width>
     <height>79</height>
    </rect>
   </property>
   <widget class="QComboBox" name="semesterBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 60%);</string>
    </property>
    <item>
     <property name="text">
      <string>第一学期</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>第二学期</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="yearBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>77</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 60%);</string>
    </property>
    <item>
     <property name="text">
      <string>一年级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>二年级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>三年级</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>四年级</string>
     </property>
    </item>
   </widget>
   <widget class="QToolButton" name="getLessonButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>129</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>129</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton#getLessonButton{background-color: rgba(255, 255, 255, 0);border:2px solid white;border-radius:10px;} QToolButton#getLessonButton:hover{ background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;} QToolButton#getLessonButton:pressed{background-color: rgba(255, 255, 255, 150);border:2px solid white;border-radius:10px;} </string>
    </property>
    <property name="text">
     <string>获取课程</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="spiderMsg">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>94</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 60%);</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="SIMS_res.qrc"/>
 </resources>
 <connections/>
</ui>
